src/main/java/com/github/timm/cucumber/generate/CucumberITGenerator.java

```diff
@@ -188,9 +186,14 @@ private String createFormatStrings() {
 
         for (int i = 0; i < formatStrs.length; i++) {
             final String formatStr = formatStrs[i].trim();
-            sb.append(String.format("\"%s:%s/%s.%s\"", formatStr,
-                            config.getCucumberOutputDir().replace('\\', '/'), fileCounter,
-                            formatStr));
+
+            if ("pretty".equalsIgnoreCase(formatStr)) {
+                sb.append("\"pretty\"");
```

Comment:
I agree, it can be a bit much, but then again, I believe this plugin should support as many Cucumber options as it can.

Yeah, I was thinking a config element for plugins directed to files in the cucumber output directory and then a config element for plugins directed to the console. I can come up with something to what might work. Give me a few days or so, I'm a bit sidetracked at work for the moment.